lua/harpeek/init.lua

```diff
@@ -75,29 +79,40 @@ local function format_item(path, index)
     end
 
     -- TODO: This logic is pretty opaque. I could probably do something nicer.
-    local format = ''
-    if Harpeek._settings.format == 'filename' then
-        format = vim.fn.fnamemodify(path, ':t') .. suffix
+    local postfix = ''
```

Comment:
Having `postfix` and `suffx` is a bit confusing to me. I think something like `line_content` or `formatted_line` would make more sense here.